### PR TITLE
Refactor component rendering structure

### PR DESCRIPTION
**Pull request by NiciP**
#### Description
- Create hompepage component
- Add switch route
- Change rendering flow 

#### What I have learned working on this feature: 
Compartmentalizing components make it more efficient to work with

### CODE REVIEW

File: src/components/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		}
+	}
+
+	render() {
+		return <>{this.isUserAuthenticated()}</>;
+	}
+}
+export default HomePage;
+
+// <div style={{ overflow: 'hidden' }}>
```
Comment (aleciabenjamin) [inline src/components/HomePage.js]: Is there a reason why comments aren't removed?

File: src/components/Menubar.js [hunk 2]
```diff
@@ -60,9 +58,9 @@ class Menubar extends Component {
 						/>
 					</div>
 
-					<p id="login_message" style={{ color: '#354151', textAlign: 'center' }}>
+					{/* <p id="login_message" style={{ color: '#354151', textAlign: 'center' }}>
```
Comment (aleciabenjamin) [inline src/components/Menubar.js]: Are the comments meant to be here?

File: src/components/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+{
+	/* <div>
+<Menubar style={{ zIndex: "10" }} user={user} />
```
Comment (Joberg95) [inline src/components/HomePage.js]: remove commented out code

File: src/components/Menubar.js (hunk 2 above)
Comment (Joberg95) [inline src/components/Menubar.js]: remove commented out code

File: src/index.js [hunk 4]
```diff
@@ -4,6 +4,14 @@ import App from './App';
...
+import { BrowserRouter } from 'react-router-dom'
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+ReactDOM.render((
+    <BrowserRouter>
+        <App/>
+    </BrowserRouter>
+    ),
+    document.getElementById('root')
```
Comment (Joberg95) [inline src/index.js]: formatting

File: src/components/HomePage.js (hunk 3 above)
Comment (NiciP) [reply src/components/HomePage.js]: done

File: src/components/Menubar.js (hunk 2 above)
Comment (NiciP) [reply src/components/Menubar.js]: done

**Review by Joberg95 — APPROVED** [review]

File: src/index.js (hunk 4 above)
Comment (NiciP) [reply src/index.js]: done

**Review by aleciabenjamin — APPROVED** [review]

**Pull request merged**